web/src/user_profile.js

```diff
@@ -284,6 +355,45 @@ export function register_click_handlers() {
         e.preventDefault();
     });
 
+    $("body").on("click", "#user-profile-modal .add-subscription-button", (e) => {
+        e.preventDefault();
+        const stream_id = Number.parseInt(user_profile_subscribe_widget.value(), 10);
+        const sub = sub_store.get(stream_id);
+        const $alert_box = $("#user-profile-streams-tab .stream_list_info");
+        function addition_success(data) {
+            if (Object.keys(data.subscribed).length > 0) {
+                ui_report.success(
+                    $t_html({defaultMessage: "subscribed successfully!"}),
```

Comment:
This should probably use sentence case; I'm not sure why this doesn't fail the capitalization linter.

Reply:
Fixed, thanks for noticing